src/main/java/uk/ac/ebi/ampt2d/metadata/SpringDataRestConfig.java

```diff
@@ -32,24 +32,14 @@
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
 import uk.ac.ebi.ampt2d.metadata.aop.StudyDeprecationAspect;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.Analysis;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.ReferenceSequence;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.File;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.Sample;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.Taxonomy;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.WebResource;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.*;
```

Comment:
In our Java coding style guide, we specifically disallow wildcard import: https://www.ebi.ac.uk/seqdb/confluence/display/VAR/Java+coding+style#Javacodingstyle-Importstatements

You can disable the wildcard import automatically imposed by the IDE like the answers in this SO issue: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Same for other places.